extensions/wikia/DesignSystem/scripts/DesignSystemGlobalNavigationOnSiteNotifications.view.js

```diff
@@ -120,28 +120,43 @@ define('ext.wikia.design-system.on-site-notifications.view', [
 						if (!avatar.avatarUrl) {
 							avatar.avatarUrl = avatarPlaceholder;
 						}
-						avatar.profileUrl = '/wiki/User:' + avatar.name;
+						avatar.profileUrl = window.wgArticlePath.replace('$1', 'User:' + avatar.name);
```

Comment:
will `window. wgArticlePath` always be present?

Reply:
Yes. And if not, then _a lot_ of JS will break.